alf/algorithms/diayn_algorithm.py

```diff
@@ -29,37 +30,39 @@
 class DIAYNAlgorithm(Algorithm):
     """Diversity is All You Need Module
 
-    This module learns a set of skill-conditional policies in an unsupervised way.
-
-    See Eysenbach et al "Diversity is All You Need: Learning Diverse Skills without a Reward Function"
+    This module learns a set of skill-conditional policies in an unsupervised
+    way. See Eysenbach et al "Diversity is All You Need: Learning Diverse Skills
+    without a Reward Function" for more details.
     """
 
     def __init__(self,
                  num_of_skills,
                  feature_spec,
                  hidden_size=256,
+                 reward_normalizer=None,
                  encoding_net: Network = None,
                  discriminator_net: Network = None,
                  name="DIAYNAlgorithm"):
         """Create a DIAYNAlgorithm.
 
         Args:
-            num_of_skills: number of skills
+            num_of_skills (int): number of skills
             hidden_size (int|tuple): size of hidden layer(s)
```

Comment:
If discriminator_net is None. A default discriminator_net with this hidden_size will be used.

Reply:
Comments added.